src/test/java/upgrade/com/atlassian/bitbucket/jenkins/internal/UpgradeTest.java

```diff
@@ -115,7 +117,13 @@ private void compareFields(Set<Class<?>> releasedClasses, Map<String, Class<?>>
                 Field releasedField = entry.getValue();
                 Field unreleasedField = unreleasedFields.get(entry.getKey());
                 if (unreleasedField == null) {
-                    //field was removed, this should be ok
+                    String fullFieldName = releasedField.getDeclaringClass().getName() + '.' + releasedField.getName();
```

Comment:
```suggestion
                    String fullFieldName = releasedField.getDeclaringClass().getName() + '#' + releasedField.getName();
```

I think your format is `className#FieldName` so you need to update here as well